index.js

```diff
@@ -20,7 +20,7 @@ Toolkit.run(async tools => {
 
   const commitMessage = process.env['INPUT_COMMIT-MESSAGE'] || 'ci: version bump to {{version}}'
   console.log('messages:', messages)
-  const commitMessageRegex = new RegExp(commitMessage.replace(/{{version}}/g, 'v\d\.\d\.\d'), 'ig');
+  const commitMessageRegex = new RegExp(commitMessage.replace(/{{version}}/g, 'v\\d+\\.\\d+\\.\\d+'), 'ig');
```

Comment:
@mattanglin you changed that line here: 
https://github.com/phips28/gh-action-bump-version/commit/c87380f10b7b5897dfc8e8ea8edbcc61f6db9047

can you check that PR pls if that works for you?

Reply:
yup, that was my bad with the regex. 🤦🏼  This fixes it. Sorry! 🙇🏼